spec/features/users/sign_in_spec.rb

```diff
@@ -102,28 +102,133 @@
     expect(oidc_redirect_url).to start_with service_provider.redirect_uris.first
   end
 
-  scenario 'User with gov/mil email directed to recommended PIV page' do
-    user = create(:user, :with_phone, { email: 'example@example.gov' })
+  context 'use_fed_domain_class set to true' do
+    let!(:federal_email_domain) { create(:federal_email_domain, name: 'gsa.gov') }
 
-    visit new_user_session_path
-    fill_in_credentials_and_submit(user.email, user.password)
-    fill_in_code_with_last_phone_otp
-    click_submit_default
-    expect(page).to have_current_path(login_piv_cac_recommended_path)
-    click_button(t('two_factor_authentication.piv_cac_upsell.add_piv'))
-    expect(page).to have_current_path(setup_piv_cac_path)
+    before do
+      allow(IdentityConfig.store).to receive(:use_fed_domain_class).and_return(true)
+    end
+
+    scenario 'User with valid fed email directed to recommend page and get to setup piv' do
```

Comment:
Should these be removed now that we have `spec/features/sign_in/piv_recommended_after_sign_in_spec.rb` ?